### PR TITLE
LPS-118086 Crypto Hash API

### PR DESCRIPTION
**Pull request by csierra**
/cc @arthurchan35 @martamedio

Hi Brian, 

this is the new version of the hashing api with the naming that you suggested to Arthur and without the builders or other artifacts that we had to form a library. 
In this all the wiring and initialization have been moved to OSGi. 

Bests. 
Carlos.

### CODE REVIEW

File: modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+crypto-hash-processor=Crypto Hash Processor
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties:1]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by csierra** [comment]
ci:forward

**Comment by csierra** [comment]
ci:stop

**Pull request closed without merging**